### PR TITLE
Feature/ab#554 enable different configs

### PR DESCRIPTION
**Pull request by witchpou**
Enable different AppTemplate config files

## Description

As we need different variants of templates for mariadb and postgres, we need a way to exchange a part of the template files.

## Motivation and Context

Using different databases in one template.

## How has this been tested?

Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx [hunk 1]
```diff
@@ -113,29 +114,30 @@ function AppTemplateCard(props) {
...
+                <Divider />
                 <CardContent>
-                    <Grid container spacing={0} alignItems={"center"}>
+                    <Grid container spacing={0} alignItems={"left"}>
                         <Grid item sm={7}>
                             <Stack spacing={2}>
                                 <Typography variant="body2" color="text.secondary">
-                                    {appTemplate.description}
+                                    <Box>{appTemplate.description}</Box>
+                                    <Box>{t("appTemplate.configFile") + ": " + appTemplate.configFile}</Box>
```
Comment (DerCed) [inline webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx:132]: What do you want to accomplish here? I think a box represents a div container, which is not nice in a typography?

**Comment by witchpou** [comment]
you can make it better...suggest sonething

File: webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx (hunk 1 above)
Comment (witchpou) [reply webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx:132]: I exactly want to have div-container ;-) Du kannst gern andere Vorschläge machen. Alles andere sah bei mir nicht schön aus.

Comment (witchpou) [reply webclient/app/src/app/features/appTemplates/appTemplateCard/AppTemplateCard.jsx:132]: done

**Pull request merged**